index.html

```diff
@@ -302,69 +351,127 @@ <h3 class="information__heading">Контакты</h3>
+            <li class="main-footer__item">
+              <a href="#" class="main-footer__link">Материалы</a>
+            </li>
+            <li class="main-footer__item">
+              <a href="#" class="main-footer__link">Техника</a>
+            </li>
+            <li class="main-footer__item">
+              <a href="#" class="main-footer__link">Инструмент</a>
+            </li>
+            <li class="main-footer__item">
+              <a href="#" class="main-footer__link">Спецпредложения</a>
+            </li>
+          </ul>
+        </div>
 
-      <div class="main-footer__middle">
-        <p class="main-footer__contacts">
-          г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8
-          <a href="tel:+78125550555" class="main-footer__phone">+7 (812) 555-05-55</a>
-        </p>
-        <ul class="main-footer__cat-list">
-          <li class="main-footer__cat-item">
-            <a href="#" class="main-footer__cat-link">Материалы</a>
-          </li>
-          <li class="main-footer__cat-item">
-            <a href="#" class="main-footer__cat-link">Техника</a>
-          </li>
-          <li class="main-footer__cat-item">
-            <a href="#" class="main-footer__cat-link">Инструмент</a>
-          </li>
-          <li class="main-footer__cat-item">
-            <a href="#" class="main-footer__cat-link">Спецпредложения</a>
-          </li>
-        </ul>
       </div>
 
       <div class="main-footer__bottom">
-        <p class="main-footer__copyright">
-          &copy; 2010–2019 Компания «Техномарт» <br>
-          Все права защищены
-        </p>
-        <p class="main-footer__social">
-          <a href="#" class="social social--vk" aria-label="Вконтакте"></a>
-          <a href="#" class="social social--fb" aria-label="Facebook"></a>
-          <a href="#" class="social social--insta" aria-label="Instagram"></a>
-        </p>
-        <p class="main-footer__feedback">
-          Обратная связь:
-          <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
-        </p>
-        <p class="main-footer__developer">
-          Разработано —
-          <a href="https://htmlacademy.ru">htmlacademy.ru</a>
-        </p>
+        <div class="wrapper">
+          <p class="main-footer__copyright">
+            &copy; 2010–2019 Компания «Техномарт» <br>
+            Все права защищены
+          </p>
+          <p class="main-footer__social">
+            <a href="#" class="social social--vk" aria-label="Вконтакте"></a>
+            <a href="#" class="social social--fb" aria-label="Facebook"></a>
+            <a href="#" class="social social--insta" aria-label="Instagram"></a>
+          </p>
+          <p class="main-footer__feedback">
+            Обратная связь:
+            <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
+          </p>
+          <p class="main-footer__developer">
+            Разработано —
+            <a href="https://htmlacademy.ru">htmlacademy.ru</a>
```

Comment:
ссылки на сторонние ресурсы лучше открывать в новой вкладке
```suggestion
            <a href="https://htmlacademy.ru" target="_blank">htmlacademy.ru</a>
```

Reply:
+